pkg/mkimage-raw-efi/make-raw

```diff
@@ -80,6 +80,8 @@ CONF_PART_SIZE=$((1024 * 1024))
 WIN_INVENTORY_PART_SIZE=$((40240 * 1024))
 # installer system partition size in bytes
 INSTALLER_SYS_PART_SIZE=$(( 300 * 1024 * 1024 ))
+# imx8 boot blob size in bytes
+IMX8_BLOB_SIZE=$(( 5 * 1024 * 1024 ))
```

Comment:
Hard code the boot blob size here it's not a good idea. It might easily change if you include OPTEE-OS or use other i.MX8MP platforms. I've seen in other BSPs that the total size recommend is 8MB. So you could read the actual size of boot blob binary in the script and set the proper size (rounding up when needed), or just set to the maximum 8MB, waste 2 or 3MB of the SD Card will not hurt if we can avoid future problems (flash the bootloader incompletely)...

Reply:
We can just increase this size to 8Mb, if at the same time we fit into the limited size of the image. Need to check.

Reply:
Seems it's OK, rebased

Reply:
I tested, everything is fine.